src/main/java/teammates/ui/controller/ActionResult.java

```diff
@@ -53,7 +53,17 @@ public ActionResult(
      * execution of the action. Messages are separated by {@code '<br>'}
      */
     public String getStatusMessage() {
-        return StringHelper.toString(statusToUser, "<br />");
+        List<String> statusMessageTexts = new ArrayList<String>();
+        
+        for (StatusMessage msg : statusToUser) {
+            statusMessageTexts.add(msg.getText());
+        }
+        
+        return StringHelper.toString(statusMessageTexts, "<br />");
+    }
+    
+    public String getStatusMessageColor() {
+        return (statusToUser == null || statusToUser.isEmpty()) ? "info" : statusToUser.get(0).getColor();
```

Comment:
In case there are multiple status messages, are you assuming all status messages will be of same importance? That may not be the case. In that case, do we show each message in the corresponding color, or use the one with the highest importance?